src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import BootstrapBreadcrumbItem from 'react-bootstrap/BreadcrumbItem'
+
+interface Props {
+  /** The children to render */
+  children?: React.ReactNode
+  /** Adds active class and renders wraps children in span */
+  active?: boolean
+  /** Adds custom event */
+  onClick?: (event: React.MouseEvent) => void
+  /** Renders anchor tag and pass href. Do not use  when active="true" */
+  href?: string
```

Comment:
we can remove href since we will never use it since we will be using something like react-router.